app/assets/javascripts/components/case/CaseForm.jsx

```diff
@@ -0,0 +1,255 @@
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="date"
+                            title="Initial Invoice Date"
+                            placeholder="Initial Invoice Date"
+                            name="initial_invoice_date"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="date"
+                            title="Last Invoice Date"
+                            placeholder="Last Invoice Date"
+                            name="last_invoice_date"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="date"
+                            title="Date Rec Initial Disbursement"
+                            placeholder="Date Rec Initial Disbursement"
+                            name="date_rec_initial_disbursement"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="text"
+                            title="Case Tracking"
+                            placeholder="Case Tracking"
+                            name="case_tracking"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="text"
+                            title="Program"
+                            placeholder="Program"
+                            name="program"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="text"
+                            title="Judge Assigned"
+                            placeholder="Judge Assigned"
+                            name="judge_assigned"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="text"
+                            title="Trial Attorney"
+                            placeholder="Trial Attorney"
+                            name="trial_attorney"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="text"
+                            title="Case Progress"
+                            placeholder="Case Progress"
+                            name="case_progress"
+                            initData={null}
+                            update={this._update} />
+                    <Input
+                            type="date"
+                            title="Date Biometrics Done"
+                            placeholder="Date Biometrics Done"
```

Comment:
for this (and other date-related inputs), can you make the placeholder an example of the date input format so there isn't ambiguity for attorneys when they need to fill this field in